TwoNote/dependencies.gradle

```diff
@@ -57,9 +57,11 @@ ext {
 
     //Microsoft dependencies version
     dualScreenLayoutVersion = "1.0.0-alpha01"
+    fluentVersion = "1.1.33"
 
     microsoftDependencies = [
-            dualScreenLayout: "com.microsoft.device:dualscreen-layout:$dualScreenLayoutVersion"
+            dualScreenLayout: "com.microsoft.device:dualscreen-layout:$dualScreenLayoutVersion",
+            fluent: "com.microsoft.design:fluent-system-icons:$fluentVersion",
```

Comment:
what functions are we using that have a fluent dependency?

Reply:
I couldn't find a good pen icon in the Android material design library so we're using a fluent one at the moment!